pubspec.yaml

```diff
@@ -14,11 +14,13 @@ description: A new Flutter application.
 version: 1.0.0+1
 
 environment:
-  sdk: ">=2.1.0 <3.0.0"
+  sdk: ">=2.2.2 <3.0.0"
 
 dependencies:
   flutter:
     sdk: flutter
+  sqflite: ^1.1.7+1
+  path: 
```

Comment:
why is path left blank?

Reply:
Either it uses the latest version or it doesn't matter, I couldn't find an example that provided a version. Pretty sure all it does is join strings based on device (ios/android)